### PR TITLE
Adding Datasources to call User and Trainee API. + day 3 task(Adding …

### PR DESCRIPTION
**Pull request by pankaj-ps**
Adding Datasources to call User and Trainee API. + day 3 task(Adding queries and Resolvers)

### CODE REVIEW

File: src/module/trainee/query.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    Trainee : async (_source,args,{dataSources},info) => {
+          const dataValue = await dataSources.traineeApi.getTrainee();
+          const {data}=dataValue;
+           console.log(data);
+          return data;
+    }
+  }
+  
+  export default Query;
+
```
Comment (pinku4444) [inline src/module/trainee/query.js]: Please remove extra space

File: src/service/trainee/trainee.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+
+    willSendRequest(request) {
+        request.headers.set('authorization', this.context.token);
+    }
+
+    async getTrainee() {
+        return this.get(`trainee`);
+    }
+
```
Comment (pinku4444) [inline src/service/trainee/trainee.js]: please remove extra space

File: src/module/trainee/query.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const Query = {
+    Trainee : async (_source,args,{dataSources},info) => {
+          const dataValue = await dataSources.traineeApi.getTrainee();
+          const {data}=dataValue;
+           console.log(data);
```
Comment (pinku4444) [inline src/module/trainee/query.js]: please remove all console

File: src/module/user/query.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const Query = {
+  me: async (_source, args, { dataSources }, info) => {
+    const user = await dataSources.userApi.getUser();
+    return user.data;
+  }
+}
+
+export default Query;
+
```
Comment (pinku4444) [inline src/module/user/query.js]: Remove extra space

File: src/module/trainee/query.js (hunk 1 above)
Comment (pankaj-ps) [reply src/module/trainee/query.js]: done

File: src/service/trainee/trainee.js (hunk 2 above)
Comment (pankaj-ps) [reply src/service/trainee/trainee.js]: done

File: src/module/trainee/query.js (hunk 3 above)
Comment (pankaj-ps) [reply src/module/trainee/query.js]: done

File: src/module/user/query.js (hunk 4 above)
Comment (pankaj-ps) [reply src/module/user/query.js]: done

**Pull request merged**